public/locales/en/userHome.json

```diff
@@ -0,0 +1,6 @@
+{
+  "titleTag": "Dashboard",
+  "welcome": "Welcome",
```

Comment:
This should probably go in common.json. There's a similar string in the LoginDialog component